x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -316,18 +334,22 @@ const nestedDeepLinks: SecurityDeepLinks = {
 
 /**
  * A function that generates the plugin deepLinks
+ * @param enableExperimental ExperimentalFeatures arg
  * @param licenseType optional string for license level, if not provided basic is assumed.
+ * @param capabilities optional arg for app start capabilities
  */
 export function getDeepLinks(
+  enableExperimental: ExperimentalFeatures,
```

Comment:
if ueba ends up being the only experimental feature that gets added/removed from deep links depending on the feature flag, i will remove when ueba feature flag is lifted